### ISSUE TITLE
Cannot have empty whitelist AND blacklist

### ISSUE THREAD

**Issue by rafaeldamasceno** [aquelemiguel/parrot#228]
### 📝 Description

If you try to have empty whitelist AND blacklist at the same time to allow all sources, the bot fills the whitelist with youtube.com, effectively only allowing YouTube as a source.

### 🪜 Reproduction Steps

1. Run `/managesources` and leave both lists empty.
2. Run `/play` with a non YouTube source
3. The "source not allowed" message is printed

### ℹ Environment / Computer Info

- **Parrot version**: ` 1.5.0`
- **Operating System**: Official Docker image

**Comment by StaticRocket** [comment]
There's an undocumented trick for this. Add "." As an entry to the blacklist.

**Comment by rafaeldamasceno** [comment]
Yeah, I do this. Anything goes for the blacklist entry and the whitelist stays empty, but it's not immediate nor intuitive 😄

**Comment by aquelemiguel** [comment]
@rafaeldamasceno Thank you for the report. Yes, this is sort of an oversight on #223. You can get around it (as you and @StaticRocket both said) but I'm still not entirely happy on how intuitive this command is. Do you (or anyone) have a proposal for a better implementation?

**Comment by mechanicstriker** [comment]
Allow by default playing music from any source.
